feature-wallet-api/src/main/java/io/novafoundation/nova/feature_wallet_api/presentation/model/AmountModel.kt

```diff
@@ -59,16 +62,23 @@ fun mapAmountToAmountModel(
     token: TokenBase,
     includeZeroFiat: Boolean = true,
     includeAssetTicker: Boolean = true,
+    formatTokenAmount: Boolean = true,
     tokenAmountSign: AmountSign = AmountSign.NONE,
     roundingMode: RoundingMode = RoundingMode.FLOOR,
     estimatedFiat: Boolean = false
 ): AmountModel {
     val fiatAmount = token.amountToFiat(amount)
 
-    val unsignedTokenAmount = if (includeAssetTicker) {
-        amount.formatTokenAmount(token.configuration, roundingMode)
+    val unsignedTokenAmount = if (formatTokenAmount) {
+        amount.formatTokenAmount(token.configuration, roundingMode, includeAssetTicker)
     } else {
-        amount.format(roundingMode)
+        val unformattedAmount = amount.format(roundingMode)
+
+        if (includeAssetTicker) {
+            unformattedAmount.withTokenSymbol(token.configuration.symbol)
+        } else {
+            unformattedAmount
+        }
```

Comment:
I do not quite get the logic here: `formatTokenAmount` calls `amount.format(roundingMode)` anyway so there is no effective difference between these if branches?

Also the task was to show **full** amount, meaning that we should format with the formatter that doesnt have abbreviations enabled